[Hyperdrive] Revisiting Learning section (#11958)
cloudflare/cloudflare-docs: content/hyperdrive/learning/query-caching.md

# Query caching

Hyperdrive automatically caches the most popular queries executed against your database, reducing the need to go back to your database (incurring latency and database load) for every query.

## What does Hyperdrive cache?

Because Hyperdrive uses database protocols, it can differentiate between a mutating query (a query that writes to the database) and a non-mutating query (a read-only query), allowing Hyperdrive to safely cache read-only queries.

> Hyperdrive determines the difference between a `SELECT` and an `INSERT`, as well as parsing the database wire-protocol and using it to differentiate between a mutating or non-mutating query.

Reviewer: ```suggestion
Besides determining the difference between a `SELECT` and an `INSERT`, Hyperdrive also parses the database wire-protocol and uses it to differentiate between a mutating or non-mutating query.
```
I think this approach conveys the original meaning a bit better.
